main.go

```diff
@@ -1,10 +1,17 @@
 package main
 
 import (
+	"os"
+
 	_ "github.com/VuKhoa23/advanced-web-be/docs"
 	"github.com/VuKhoa23/advanced-web-be/startup"
 )
 
 func main() {
+	if len(os.Args) > 1 && os.Args[1] == "migrate" {
+		startup.Migrate()
+		return
+	}
+
 	startup.Execute()
```

Comment:
I mentioned this before, please specify the command, not the number of arguments

Ex:
`if argument == migrate-up then do something else do something`

Reply:
DONT USE MAGIC NUMBER. Please be specific in your code

Reply:
i already change from "migrate" to "migrate-up", imo using len(os.Args) > 1 && os.Args[1] == "migrate" is not magic at all, it just check the 1st argument of the command line to see what its type

Reply:
using len(os.Args) > 1 && os.Args[1] == "migrate-up" is the only way to do i think so

Reply:
`if len(argument) > 1`

Do you think anyone will understand what this line of code used for ? That why I said that code block is magic code

Reply:
since until now we only have 1 command.  So maybe we can consider this as a tech debt and get back to it later

Reply:
i already add a function in main to resolve it, go check it out